emily/handler/src/api/handlers/health.rs

```diff
@@ -15,7 +15,8 @@ use crate::common::error::Error;
         (status = 404, description = "Address not found", body = ErrorResponse),
         (status = 405, description = "Method not allowed", body = ErrorResponse),
         (status = 500, description = "Internal server error", body = ErrorResponse)
-    )
+    ),
+    security(("ApiGatewayKey" = []))
```

Comment:
Do we want this to be private?

Reply:
I thought so?

Reply:
It seems the kind of endpoint that may be useful for external clients, @aldur WDYT?

Reply:
What do you think about us keeping it private for now so we don't need to worry about exposing too much info from this endpoint and then making it public later if we decide we want to?